py3status/modules/getjson.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+Query a url and display the json response
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds
+        (default 30)
+    format: placeholders will be replaced by the returned json key names
+        (default '')
+    sep: the separator between parent and child objects
+        (default '_')
```

Comment:
hmm, I don't like `sep` as a name however `separator` shouldn't be used due to its meaning in i3bar protocol.  I'm not sure what would be a better name though `delimiter`?